numpy/core/tests/french_locale.py

```diff
@@ -0,0 +1,77 @@
+"""Provide class for testing in French locale
+
+"""
+from __future__ import division, absolute_import, print_function
+
+import sys
+import locale
+
+from numpy.testing import SkipTest
+
+__ALL__ = ['FrenchLocale']
+
+
+def find_french_locale():
```

Comment:
Naming: I think the point is not to switch to specifically French locale, but to a locale where the decimal separator is comma and not period.

Reply:
Agree in general, but I chose "French" so folks would know what the locale used was. Maybe "DecimalPointAsComma" or some such?

Reply:
Or let's just keep it as "french" and not bikeshed, the docstring already seems to say what it's for.
Otherwise, this PR is LGTM.

Reply:
OK, lets leave it. If changes are needed at some point, fixing things up should be easy enough, just a few additions and name changes.

Reply:
Well, I changed things as there is also the possibility of a Finnish locale :-) Renamed the module `_locales.py` and the class to `CommaDecimalPointLocale`.